open mic summary FDOs (#195)
kbss-cvut/kbss-website: _posts/2026-04-17-fair-digital-objects-open-mic-summary.markdown

---
title: "FAIR Digital Objects"
categories: [ Open Mic Summary, Open Mic ]

> excerpt: "Showcase of a metadata management and publishing environment Piveau in the context of need for publishing and metadata discovery environment for scientific metadata described by DCAT based metadata profiles."

Reviewer: This looks like excerpt from an older Open mic.

---

On Friday, 17th April 2026, at 10:30, [Michal Med](https://kbss.felk.cvut.cz/web/team#michal-med) talked about FAIR Digital Objects as a solution to the problem of drowning in passive data. He was present at the FDO Conference in Vienna taken from 25th to 27th March 2026, which was focused on FAIR Digital Objects.

{% include video id="PQskg0m87k8" provider="youtube" %}

##### Problem definition

> The growth in the numbers of the new data sets, either collected or generated, is exponential. Those data are moreover sitting passively in the silos without being properly described or published. Scientists are spending 80 percent of the time with data by its finding and cleaning rather then with the actual analysis. 

Reviewer: ```suggestion
The growth in the numbers of the new data sets, either collected or generated, is exponential. Those data are moreover sitting passively in the silos without being properly described or published. Scientists are spending 80 percent of the time with data by its finding and cleaning rather than with the actual analysis. 
```

##### FAIR Principles

> FAIR Principles were introduces in 2016 in order to provide guidelines to improve machine-actionability (ability of computational systems to find, access, interoperate and reuse data without or with minimal human intervention) of digital assets. The letters in FAIR stand for **F**indability, **A**ccessibility, **I**nteroperability and **R**eusability.

Reviewer: ```suggestion
FAIR Principles were introduced in 2016 in order to provide guidelines to improve machine-actionability (ability of computational systems to find, access, interoperate and reuse data without or with minimal human intervention) of digital assets. The letters in FAIR stand for **F**indability, **A**ccessibility, **I**nteroperability and **R**eusability.
```

##### FAIR Digital Objects
As FAIR Principles are concepts giving the way where to go, there are no technical specifications on how to implement them. To turn the principles into the deployable architecture, the FAIR Digital Objects (FDOs) were introduced. FAIR Digital Objects are representing data as active autonomous units identified uniquely with persistent ID, described with semantically rich metadata and published in accordance with FAIR principles.

> {% include figure image_path="./assets/images/posts/2026-04-17-smart-container.png" alt="Research data jungle" %}

Reviewer: This file does not seem to exist in Git.

> Often used is a metaphor with a smart shipping container with a bar code, which can be read by any shipyard crane in the world and with a manifest telling clarly what is inside the container and how to handli it properly. 

Reviewer: ```suggestion
An often-used metaphor describes an FDO as a smart shipping container with a bar code, which can be read by any shipyard crane in the world and with a manifest telling clearly what is inside the container and how to handle it properly. 
```
